addons/edi/models/edi_document.py

```diff
@@ -12,6 +12,11 @@
 
 AutodetectDocument = namedtuple("AutodetectDocument", ["type", "inputs"])
 
+PROCESSING_STATES = [
```

Comment:
Nit: Indentation doesnt look right, maybe there are tabs and spaces mixed

Reply:
Updated.